config/configuration.yml.example

```diff
@@ -222,6 +222,13 @@ default:
   #   take control over the given path to create and delete repositories directly
   #   when created in the frontend.
   #   NOTE: Disabling :managed repositories using disabled_types takes precedence over this setting.
+  # mode:
+  #   The file mode to set the repository folder to (defaults to 0700).
+  #   Important: The user running OpenProject must be a member of this group in order to have the
+  #   privilege to set the group ownership.
```

Comment:
Shouldn't the "Important" comment belong to the `group` attribute, instead of the `mode` attribute?

Maybe I am reading it wrong oO